app/logic/solana_logic.rb

```diff
@@ -31,9 +31,9 @@ def batch_touch
       batch = Batch.where(uuid: p.payload[:batch_uuid]).first
 
       # IMPORTANT: Do not use this values on index page, it's only for show.
-      skipped_slot_percent_moving_average = ValidatorBlockHistory.average_skipped_slot_percent_for(
+      skipped_slot_percent_moving_average = Stats::ValidatorBlockHistory.new(
```

Comment:
```suggestion
      moving_average_skipped_slot_percent = Stats::ValidatorBlockHistory.new(
```
I think we could update variables names similar to the method names from Stats, it looks less confused to me.